docs/plugins/preserve-tombs.rst

```diff
@@ -0,0 +1,22 @@
+preserve-tombs
+==============
+
+.. dfhack-tool::
+    :summary: Fix tombs being unassigned to units on death
+    :tags: fort bugfix
+
+If you find that the tombs you assign to units get unassigned from them when
+they die (e.g. your nobles), this tool can help fix that.
+
+Usage
+-----
+
+::
+
+    enable preserve-tombs
+    preserve-tombs status
```

Comment:
I suggest printing status when no options are passed as well, that is, changing the syntax to `preserve-tombs [status]`